src/test/java/io/logz/guice/jersey/filters/JettyTestFilter.java

```diff
@@ -0,0 +1,33 @@
+package io.logz.guice.jersey.filters;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class JettyTestFilter implements Filter {
```

Comment:
Maybe AddHeaderJettyFilter

Reply:
The header was just the easiest way I could actually assert that this filter was executed and registration worked, it's not really the point of the class, and I think changing the class name might give the wrong impression

Reply:
I think that when you read the test, and you see the name of the filter, it's easier for you to understand the Tester lambda written there, and save you the trouble of jumping into something generically called JettyTestFilter and read its code to figure out it's just adding a header

Reply:
Done :)